package.json

```diff
@@ -66,7 +66,7 @@
     "langly": "0.0.14",
     "normalize.css": "^8.0.1",
     "stencil-router-v2": "^0.6.0",
-    "tidily": "0.0.47"
+    "tidily": "qijieye/tidily#master"
```

Comment:
Is this correct?

Reply:
Yep, this PR is waiting for that branch to be merged
https://github.com/payfunc/tidily/pull/54
![image](https://user-images.githubusercontent.com/14332757/147242674-53a9913c-7c4a-4fc8-a7a7-09708e86b613.png)